sql/mob_pools.sql

```diff
@@ -3245,7 +3245,7 @@ INSERT INTO `mob_pools` VALUES (3184,'Poroggo_Prince','Poroggo_Prince',196,0x000
 INSERT INTO `mob_pools` VALUES (3185,'Poroggo_Seducteur','Poroggo_Seducteur',196,0x0000130700000000000000000000000000000000,4,4,12,240,100,0,1,1,1,0,0,0,0,0,0,0,2,1,0,196);
 INSERT INTO `mob_pools` VALUES (3186,'Poroggo_Servant','Poroggo_Servant',196,0x0000BF0800000000000000000000000000000000,1,1,0,240,100,0,0,0,0,0,0,0,0,3,0,0,0,1,0,196);
 INSERT INTO `mob_pools` VALUES (3187,'Poroggo_s_Toady','Poroggo_s_Toady',196,0x0000BF0800000000000000000000000000000000,1,1,0,240,100,0,0,0,0,0,0,0,0,0,0,0,0,1,0,196);
-INSERT INTO `mob_pools` VALUES (3188,'Porphyrion','Porphyrion',126,0x0000830200000000000000000000000000000000,11,1,1,240,100,0,1,0,1,2,0,32,456,159,0,0,0,1,0,126);
+INSERT INTO `mob_pools` VALUES (3188,'Porphyrion','Porphyrion',126,0x0000830200000000000000000000000000000000,11,11,5,240,100,0,1,0,1,2,0,32,456,159,0,0,0,1,0,126);
```

Comment:
For future reference:
The cmbSkill column does wrong headed things and should even exist really, mobs don't really have the attack type their model might lead one to believe (mimas isn't really hitting you with an axes damage type) but for the moment it can cause strange behavior like dragoons multi hitting, so I recommend leaving it be in the future. The subjob, I don't know what this mob should have there. I suspect that many retail mobs default to a war sub. Unlike players they don't get "non sub" from job changing to the same main and sub at a moogle. In this case I'm merging as the rng/rng you set here.